chalice/app.py

```diff
@@ -1846,7 +1817,7 @@ def create_request_object(self, event: Any,
         return None
 
     def __call__(self, event: Any, context: Any) -> Any:
-        def wrapped_event(request: Request) -> Response:
+        def wrapped_event(unused: Request) -> Response:
```

Comment:
I'd prefer to minimize changes here.  Even though the arg is not being used, we don't really need to rename this arg.